test/libsolidity/syntaxTests/inlineAssembly/storage_reference_empty_offset.sol

```diff
@@ -1,9 +1,9 @@
 contract C {
     function f() public pure {
         assembly {
-            _offset
+            let x := _offset
```

Comment:
Should the error here actually just be ``_offset not found`` and the interesting case is ``let _offset := ...``?

Reply:
I think if we want to change that, it should be a separate PR for improving error messages (not here)?

Reply:
Even if we stick to the somewhat misleading error messages here for now, we should still have a test for ``let _offset := ...`` here as well. As far as I can see this would be the case the error message was actually meant for and I'm not sure it's handled properly after the changes. (At the very least we should check how this behaves and create a new issue for cleaning this behavior up after this PR, if needed)

Reply:
Created issue: https://github.com/ethereum/solidity/issues/7576